README.md

```diff
@@ -30,8 +25,7 @@ Plugins are installed into the Observability Portal product as part of the produ
 ## Execution Environments
 
 A plugin may be executed in a cloud environment as part of the Observability Portal product or on a machine within a customer's premises (upon which the customer has installed a Relay Agent). If both environments are equally suitable for a plugin, it can
```

Comment:
```suggestion
A plugin may be executed within SquaredUp's SaaS environment or on a machine within a customer's premises (upon which the customer has installed a Relay Agent). If both environments are equally suitable for a plugin, it can
```